plugins/inject-base.js

```diff
@@ -0,0 +1,7 @@
+const origin = process.env.SITE_HOST ||
+    (typeof (window) !== 'undefined' ? window.location.origin : '/');
+const base = origin + (origin.endsWith('/') ? '' : '/');
```

Comment:
```suggestion
const base = origin.trim().replace(/\/*$/, '') + '/';
```
Ensure we always have a single trailing slash